snippets/language-svg.cson

```diff
@@ -1,101 +1,352 @@
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/defs'
   'desc':
     'prefix': 'de'
     'body': '<desc>$1</desc>'
+    'description': 'Each container element or graphics element in an SVG drawing can supply a desc description string where the description is text-only. When the current SVG document fragment is rendered as SVG on visual media, desc elements are not rendered as part of the graphics. Alternate presentations are possible, both visual and aural, which display the desc element but do not display path elements or other graphics elements. The desc element generally improve accessibility of SVG documents'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/desc'
   'ellipse':
     'prefix': 'e'
     'body': '<ellipse cx="${1:50}" cy="${2:50}" rx="${3:20}" ry="${4:30} fill="${5:#CC0F16}" />'
+    'description': 'The ellipse element is an SVG basic shape, used to create ellipses based on a center coordinate, and both their x and y radius. Ellipses are unable to specify the exact orientation of the ellipse (if, for example, you wanted to draw an ellipse tilted at a 45 degree angle), but can be rotated by using the transform attribute.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/ellipse'
+  'feColorMatrix':
+    'prefix': 'feColorMatrix'
+    'body': '<feColorMatrix in="${1:SourceGraphic}" type="${2:saturate}" values="${3:0.2}" />'
+    'description': 'This filter changes colors based on a transformation matrix. Every pixel\'s color value (represented by an [R,G,B,A] vector) is matrix multiplied to create a new color.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feColorMatrix'
+  'feComposite':
+    'prefix': 'feComposite'
+    'body': '<feComposite in="${1:SourceGraphic}" in2="${2:light}" operator="${3:arithmetic}" k1="${4:1}" k2="${5:0}" k3="${6:0}" k4="${7:0}"/>'
+    'description': 'This filter primitive performs the combination of two input images pixel-wise in image space using one of the Porter-Duff compositing operations: over, in, atop, out, xor. Additionally, a component-wise arithmetic operation (with the result clamped between [0..1]) can be applied.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feComposite'
+  'feDiffuseLighting':
+    'prefix': 'feDiffuseLighting'
+    'body': '<feDiffuseLighting in="${1:SourceGraphic}" result="${2:light}" lighting-color="${3:white}"><fePointLight x="$4" y="$5" z="$6" /></feDiffuseLighting>'
```

Comment:
I don't think it's a good idea to default the 'result' attributes, as that's bound to lead to naming conflicts.